tiledbimg/converters/ome_zarr.py

```diff
@@ -1,24 +1,134 @@
+import glob
+import os
+import pickle
+from dataclasses import dataclass
+from typing import Any, Dict, Sequence
+
 import numpy as np
+import tiledb
+
+# import xmltodict
 import zarr
+from numcodecs import Blosc
+from ome_zarr.reader import Node, Reader, ZarrLocation
+from ome_zarr.writer import write_multiscale
+
+from .base import ImageConverter, ImageReader, ImageWriter
+
+
+@dataclass(frozen=True)
+class Level:
+    node: Node
+
+
+class OMEZarrWriter(ImageWriter):
+    def __init__(self, input_path: str, output_path: str):
```

Comment:
Main reason for that is that `from_tiledb` and `to_tiledb` live under `base.py`. As base functions should be as generic as they can be. We would have the same problem of carrying the output path in `to_tilebd` as well if our target was not always Tiledb, thus able to create the TLDB group inside the `to_tiledb`. I could create a `get_output_group` same as `_get_image_writer` under `ImageWriter` and move there the group creation
```
 self._output_group = zarr.group(
            store=zarr.storage.DirectoryStore(path=output_path), overwrite=True
        )
```
Since we haven't concluded what will be the final design for OME-TIFF and OME-Zarr and what shared code can they utilise. I thought that safest plan is to carry it in the OME-Zarr-Writer constructor.